components/typography/typography.css

```diff
@@ -0,0 +1,106 @@
+@import "../design-guide.css";
+
+* {
+  box-sizing: border-box;
+}
+
+h1#h1 {
```

Comment:
using class will be good instead of id

Reply:
It was conflicting with my documentation page specificity so I used ID. 
For now, I will keep it like this but will think about how to remove it.